docker-image/v1.1/debian-logzio/conf/fluent.conf

```diff
@@ -11,26 +11,27 @@
   endpoint_url "https://listener.logz.io:8071?token=#{ENV['LOGZIO_TOKEN']}&type=#{ENV['LOGZIO_LOGTYPE']}"
   output_include_time true
   output_include_tags true
-
-  # Set the buffer type to file to improve the reliability and reduce the memory consumption
-  buffer_type file
-  buffer_path /var/log/fluentd-buffers/stackdriver.buffer
-  # Set queue_full action to block because we want to pause gracefully
-  # in case of the off-the-limits load instead of throwing an exception
-  buffer_queue_full_action block
-  # Set the chunk limit conservatively to avoid exceeding the GCL limit
-  # of 10MiB per write request.
-  buffer_chunk_limit 2M
-  # Cap the combined memory usage of this buffer and the one below to
-  # 2MiB/chunk * (6 + 2) chunks = 16 MiB
-  buffer_queue_limit 6
-  # Never wait more than 5 seconds before flushing logs in the non-error case.
-  flush_interval 5s
-  # Never wait longer than 30 seconds between retries.
-  max_retry_wait 30
-  # Disable the limit on the number of retries (retry forever).
-  # disable_retry_limit
-  # Use multiple threads for processing.
-  num_threads 2
+  <buffer>
+    # Set the buffer type to file to improve the reliability and reduce the memory consumption
+    @type file
+    path /var/log/fluentd-buffers/stackdriver.buffer
+    # Set queue_full action to block because we want to pause gracefully
+    # in case of the off-the-limits load instead of throwing an exception
+    overflow_action block
+    # Set the chunk limit conservatively to avoid exceeding the GCL limit
+    # of 10MiB per write request.
+    chunk_limit_size 2M
+    # Cap the combined memory usage of this buffer and the one below to
+    # 2MiB/chunk * (6 + 2) chunks = 16 MiB
+    queued_chunks_limit_size 6
```

Comment:
`queued_chunks_limit_size` is not same as `buffer_queue_limit`.

https://docs.fluentd.org/v1.0/articles/api-plugin-helper-compat_parameters

Reply:
Thanks! I thought I had matched up everything right, but I missed that one.